torch/_inductor/codegen/common.py

```diff
@@ -1052,26 +1059,40 @@ def indirect_load(self, name: str, index: sympy.Expr):
         finally:
             self.loads = prior
 
-    def store_reduction(self, name, index, value):
+    def store_reduction(self, name: str, index: sympy.Expr, value: CSEVariable):
         raise NotImplementedError()
 
-    def store(self, name, index, value, mode=None):
+    def store(
+        self, name: str, index: sympy.Expr, value: CSEVariable, mode: StoreMode = None
+    ):
         raise NotImplementedError()
 
-    def reduction(self, dtype, src_dtype, reduction_type, value):
+    def reduction(
+        self,
+        dtype: torch.dtype,
+        src_dtype: torch.dtype,
+        reduction_type: ReductionType,
+        value: CSEVariable,
```

Comment:
For `reduction_type="welford_combine"` value is a tuple `CSEVariable`
```suggestion
        value: Union[CSEVariable, Tuple[CSEVariable, ...]],
```